### ISSUE TITLE
Unicode error - create torrent dialog [Windows]

### ISSUE THREAD

**Issue by xoriole** [Tribler/tribler#4881]
```python
Traceback (most recent call last):
File "TriblerGUI\dialogs\createtorrentdialog.py", line 132, in on_torrent_created
File "TriblerGUI\dialogs\createtorrentdialog.py", line 140, in add_torrent_to_channel
File "lib\encodings\utf_8.py", line 16, in decode
UnicodeEncodeError: 'ascii' codec can't encode character u'\xf1' in position 5: ordinal not in range(128)
```
Report [#27729](http://reporter.tribler.org/stacktrace/27729), [27704](http://reporter.tribler.org/stacktrace/27704), [26610](http://reporter.tribler.org/stacktrace/26610)

**Comment by qstokkink** [comment]
Seems to be the `ñ`.

**Comment by ichorid** [comment]
fixed by #4966
